src/vaults/yields.utils.ts

```diff
@@ -486,79 +368,82 @@ async function constructEmissionYieldSources(
  * @param yieldEvents events sourced from either on chain or the graph
  * @returns yield summary providing the base information for construction of yield sources
  */
-async function evaluateYieldEvents(
-  chain: Chain,
-  vault: VaultDefinitionModel,
-  yieldEvents: VaultPerformanceItem[],
-): Promise<VaultYieldSummary> {
-  const sdk = await chain.getSdk();
-  const harvestReport: HarvestReport[] = [];
+async function evaluateYieldEvents(chain: Chain, vault: VaultDefinitionModel): Promise<VaultYieldSummary> {
+  const yieldEvents = await queryVaultYieldEvents(chain, vault);
 
-  let totalHarvested = 0;
-  let totalVaultPrincipal = 0;
-  const tokenEmissionAprs = new Map<string, number>();
+  /**
+   * ON 8/15 AN INCORREC PROCESSING OF A BADGER REWARDS PROCESSOR OCCURED.
+   * AS A RESULT, THE EVENTS INCLUDED IN TREE DISTRIBUTIONS DID NOT MAKE IT
+   * TO THE GRAPH, OR ANY SOURCE OF ON CHAIN DATA CURRENTLY SUPPORTED BY
+   * THE CURRENT YIELD SYSTEM.
+   *
+   * https://etherscan.io/tx/0x1e3e7c71012d36e936b768a37e9784125a00f205a22bd808f045968a506bb1ce#eventlog
+   *
+   * THIS TRANSACTION CONTAINS THE SINGLE BADGER TREE DISTRIBUTION WE ARE
+   * ALLOCATING TO GRAVI_AURA VAULT AS A MISSED - AND NOT CAPTURED SOURCE.
+   *
+   * THIS CODE SHOULD BE REMOVED BY 08/29.
```

Comment:
removed by 8/29?

Reply:
will merge this pr and remove this later - the code actually is now filtered by the `filterInfluenceEvents` properly (confirmed yesterday) - this will need to be manually added in the vault harvests capture code though! so it will be like a manual patch into that process - i imagine there might be some issues like this in the past as well but unfortunately i don't think they are documented / captured. we should probably make an easy way to register any issue like this for future but thats out of scope